components/MapContainer.vue

```diff
@@ -84,13 +84,18 @@ const handlePinClick = (resultId: string) => {
     })
 }
 
+const handleProfessionalClick = (professionalId: string) => {
```

Comment:
I don't think this is supposed to be in the map container. It should be from the searchResultDetails